### ISSUE TITLE
how to see catboost biuiltin pictures for running code not on jupyter notebooks

### ISSUE THREAD

**Issue by Sandy4321** [catboost/catboost#747]
Problem: {
how to see catboost biuiltin pictures for running code not on jupyter notebooks
for example for 
https://github.com/catboost/tutorials/blob/master/classification/classification_tutorial.ipynb
model = CatBoostClassifier(
    iterations=50,
    random_seed=63,
    learning_rate=0.5,
    custom_loss=['AUC', 'Accuracy']
)
model.fit(
    X_train, y_train,
    cat_features=cat_features,
    eval_set=(X_validation, y_validation),
    verbose=False,
    plot=True
)
output is
<IPython.core.display.HTML object>
MetricVisualizer(data={'catboost_info': {'path': 'catboost_info', 'name': 'catboost_info', 'content': {'passed_iterations': 5, 'total_iterations': 50, 'data': {'iterations': [{'learn': [0.3019400853, 0.9421705131], 'iteration': 0, 'passed_time': 0.04627532499, 'remaining_time': 2.267490924, 'test': [0.3026467179, 0.5002429937, 0.9418675618]}, {'learn': [0.2107910544, 0.9421705131], 'iteration': 1, 'passed_time': 0.1502330007, 'remaining_time': 3.605592016, 'test': [0.2073447266, 0.7856287626, 0.9418675618]}, {'learn': [0.1834069853, 0.9446881556], 'iteration': 2, 'passed_time': 0.2760017308, 'remaining_time': 4.324027116, 'test': [0.174371668, 0.8184596964, 0.9423252975]}, {'learn': [0.1750651564, 0.9464047301], 'iteration': 3, 'passed_time': 0.3708500158, 'remaining_time': 4.264775182, 'test': [0.1617974528, 0.8331445508, 0.9459871834]}, {'learn': [0.1735443824, 0.9464810223], 'iteration': 4, 'passed_time': 0.4351767031, 'remaining_time': 3.916590328, 'test': [0.1586617597, 0.8338899015, 0.9475129692]}, {'learn': [0.1701381132, 0.9470150677], 'iteration': 5, 'passed_time': 0.5434966218, 'remaining_time': 3.985641893, 'test': [0.1547567505, 0.8559319584, 0.9485810192]}], 'meta': {'launch_mode': 'Train', 'name': 'experiment', 'iteration_count': 50, 'learn_metrics': [{'best_value': 'Min', 'name': 'Logloss'}, {'best_value': 'Max', 'name': 'Accuracy'}], 'test_sets': ['test'], 'test_metrics': [{'best_value': 'Min', 'name': 'Logloss'}, {'best_value': 'Max', 'name': 'AUC'}, {'best_value': 'Max', 'name': 'Accuracy'}], 'learn_sets': ['learn']}}}}}, layout=Layout(align_self='stretch', height='500px'))

but there is no picture
![image](https://user-images.githubusercontent.com/11426119/55038811-71b9da80-4ff8-11e9-967a-e6857f3f443c.png)

}
catboost version: {version}
Operating System: {OS}
CPU: {CPU}
# GPU: {GPU}

**Comment by annaveronika** [comment]
To see the picture you need to run jupyter notebook in browser, not in console.
And you also need to install widgets.

**Comment by Sandy4321** [comment]
it is the point, I need to run without jupyter notebook 
So running from the command line without jupyter notebook at all
actually also running from IDE for example pycharm, in this case no use  jupyter notebook at all

**Comment by annaveronika** [comment]
Unfortunatelly jupyter notebook visualization is not supported without jupyter notebook.
But you can use tensorboard that supports parts of this functionality. Please refer to the docs for further info.

**Comment by annaveronika** [comment]
I hope tensorboard will solve the issue.

**Comment by Sandy4321** [comment]
you want to say that I do have not installed tensorboard ?

**Comment by Sandy4321** [comment]
I do have tensorboard installed 
![image](https://user-images.githubusercontent.com/11426119/55568082-fda0c600-56cc-11e9-98f5-1c27d24227e0.png)

**Comment by Sandy4321** [comment]
it is what I get for pip list
![image](https://user-images.githubusercontent.com/11426119/55568208-48bad900-56cd-11e9-8078-b9ad64a27e9c.png)

**Comment by Sandy4321** [comment]
or do you mean I need to do something like this https://stackoverflow.com/questions/33634008/how-do-i-install-tensorflows-tensorboard

**Comment by annaveronika** [comment]
Have you read documentation for visualization?

**Comment by annaveronika** [comment]
https://catboost.ai/docs/features/visualization_tensorboard.html - just in case if you have not seen it yet.

**Comment by Sphinxs** [comment]
@annaveronika Even after I installed `ipywidgets` and started the notebook on browser I receive `MetricVisualizer(layout=Layout(align_self='stretch', height='500px'))`  instead of the image itself. I have set both `verbose=False` and `plot=True`. I'm using version `0.24.2`.

**Comment by atercygnus** [comment]
Have the same problem as @Sphinxs ( does not work for me in Jupyter lab even with ipywidgets...
How to localize a problem?

**Comment by Sphinxs** [comment]
> Have the same problem as @Sphinxs ( does not work for me in Jupyter lab even with ipywidgets...
> How to localize a problem?

I fixed it by uninstalling Jupyter Lab and installing it again, besides Tensor Board manually, then I just followed the steps that I describe above and it works.

**Comment by VANESSAL1U** [comment]
> > Have the same problem as @Sphinxs ( does not work for me in Jupyter lab even with ipywidgets...
> > How to localize a problem?
> 
> I fixed it by uninstalling Jupyter Lab and installing it again, besides Tensor Board manually, then I just followed the steps that I describe above and it works.

Same problem. I uninstall jupyter and re-install it after seeing your reply, it works... unbelievable

**Comment by garikhgh** [comment]
hi there, 
I run catboost regressor on GPU RTX 3060 12 GB and the data is 28000x160. after 1000 iteration when the early stoping is satisfied the kernel of jupyter died. is it possible to ran the model in vs code or pycharm, or what to do to keep the kernel alive?
